weblate/trans/tests/utils.py

```diff
@@ -188,6 +194,8 @@ def _create_component(
             raise SkipTest(f"File format {file_format} is not supported!")
         if "project" not in kwargs:
             kwargs["project"] = self.create_project()
+        if "category" not in kwargs:
+            kwargs["category"] = self.create_category(project=kwargs["project"])
```

Comment:
It is okay for category to be not set. If you need it for a specific test, just create it there.

```suggestion
```